### PR TITLE
Auth: Add Owner information to DB

### PR DESCRIPTION
**Pull request by daltonfury42**
Pretty naive approach, will have to think of better ways of doing this.

### CODE REVIEW

File: complete/src/main/java/com/example/restservice/controller/advices/SecurityAdvice.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+@ControllerAdvice
+public class SecurityAdvice extends RequestBodyAdviceAdapter {
+
+  private final JwkProvider provider;
+  private final LoggedInUserInfo loggedInUserInfo;
+
+  @Autowired
+  SecurityAdvice(LoggedInUserInfo loggedInUserInfo) throws MalformedURLException {
+    this.loggedInUserInfo = loggedInUserInfo;
+    String keyUrl = "https://cognito-idp.ap-southeast-1.amazonaws.com/ap-southeast-1_iQdl5AVrA/.well-known/jwks.json";
```
Comment (thehamzarocks) [inline complete/src/main/java/com/example/restservice/controller/advices/SecurityAdvice.java:31]: Hmm might need a better url later?

File: complete/src/main/java/com/example/restservice/dao/Queue.java [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 import org.hibernate.annotations.GenericGenerator;
 
 @Entity
 @Table(name = "queue")
+@Getter
```
Comment (thehamzarocks) [inline complete/src/main/java/com/example/restservice/dao/Queue.java:18]: Hmm I thought the Lombok changes were already done?

Comment (daltonfury42) [reply complete/src/main/java/com/example/restservice/dao/Queue.java:18]: This one was missed

File: complete/src/main/java/com/example/restservice/controller/advices/SecurityAdvice.java (hunk 1 above)
Comment (daltonfury42) [reply complete/src/main/java/com/example/restservice/controller/advices/SecurityAdvice.java:31]: this is fully internal, so would be fine.

Comment (thehamzarocks) [reply complete/src/main/java/com/example/restservice/controller/advices/SecurityAdvice.java:31]: Maybe keep this in a constants file then?

**Pull request merged**